common/src/main/java/whocraft/tardis_refined/client/TardisClientData.java

```diff
@@ -75,6 +76,9 @@ public ResourceKey<Level> getLevelKey() {
     //Not saved to disk, no real reason to be
     private int nextAmbientNoiseCall = 40;
 
+    // Independent of the hums logic
+    private int nextFemaleAmbientNoiseCall = 12000;
```

Comment:
Make the variable name more related to its purpose, so that it is more generic meaning, incase we change the sound in the future.

Reply:
Made more specific.